.github/workflows/ci.yml

```diff
@@ -1,45 +1,29 @@
-  name: CI
```

Comment:
Changes in this file look lie indention or something, right?

Reply:
Yes, it's indentation